src/metadata/routes/tags.clj

```diff
@@ -45,14 +69,31 @@ authenticated user's tags that contain the fragment."
     (POST "/user" []
       :query [{:keys [user]} StandardUserQueryParams]
       :body [body (describe TagRequest "The user tag to create.")]
-      :return TagDetails
+      :responses {200      {:schema      TagDetails
+                            :description "The tag was successfully created"}
+                  400      {:schema      ErrorResponseBadTagRequest
+                            :description "The `value` was not unique, too long,
+                             or the request body wasn't syntactically correct"}
+                  500      {:schema      ErrorResponseUnchecked
+                            :description "Unchecked errors"}
+                  :default {:schema      ErrorResponse
+                            :description "All other errors"}}
       :summary "Create a Tag"
       :description "This endpoint creates a tag for use by the authenticated user."
       (ok (tags/create-user-tag user body)))
 
     (DELETE "/user/:tag-id" []
       :path-params [tag-id :- TagIdPathParam]
       :query [{:keys [user]} StandardUserQueryParams]
+      :coercion middleware/no-response-coercion
+      :responses {200      {:schema      nil
```

Comment:
Even though this 200 `schema` is defined as `nil` for this `DELETE` endpoint (success returns no response body), a success was still failing response validation (`nil != map`) without using `no-response-coercion`.